app/build.gradle

```diff
@@ -331,4 +331,12 @@ dependencies {
     annotationProcessor "org.projectlombok:lombok:1.18.10"
 }
 
+tasks.withType(Test) { 
+  testLogging {
+    exceptionFormat "full"
+    events "started", "skipped", "passed", "failed"
+    showStandardStreams true
+  }
+}
```

Comment:
Can you explain why this change is needed?

Reply:
Without this change, when one is running the unit tests, he gets a summery at the end of what has ahppened.
With this change, one can also see the test results as they are running.

It will now look like (also with green color for passing tests.):

com.eveningoutpost.dexdrip.watch.thinjam.utils.BitmapToolsTest > unpackMonoBytesToRGB565Test STARTED

com.eveningoutpost.dexdrip.watch.thinjam.utils.BitmapToolsTest > unpackMonoBytesToRGB565Test PASSED

com.eveningoutpost.dexdrip.watch.thinjam.utils.BitmapToolsTest > packRGB565bytesToMonoTest STARTED

com.eveningoutpost.dexdrip.watch.thinjam.utils.BitmapToolsTest > packRGB565bytesToMonoTest PASSED